pkg/kn/commands/service/create.go

```diff
@@ -303,12 +303,11 @@ func constructServiceFromFile(cmd *cobra.Command, editFlags ConfigurationEditFla
 	// Set namespace in case it's specified as --namespace
 	service.ObjectMeta.Namespace = namespace
 
-	// We need to generate revision to have --force replace working
-	revName, err := servinglib.GenerateRevisionName(editFlags.RevisionName, &service)
```

Comment:
Why has this been removed ?

Reply:
It's superseded by combination of `markFlagMakesRevision` and `editFlags.Apply` combination, per my suggestion/proposal from [previous PR](https://github.com/knative/client/pull/913#discussion_r451481222).

https://github.com/knative/client/pull/937/files#diff-74b189b0b506d256ede4322475d950f5R285

Reply:
I see, and I think it makes sense.